testsuite/classlibrary/TestServer.sc

```diff
@@ -38,14 +33,1297 @@ TestServer : UnitTest {
 		this.sync;
```

Comment:
while you are at it, could you also remove the trailing semicolon here?

Reply:
https://github.com/supercollider/supercollider/wiki/%5BWIP%5D-Code-style-guidelines#recommendation-dont-place-a-semicolon-after-the-final-statement-of-a-method-or-function

I think we agreed this is an optional style point. any particular reason to insist here?